docs/blog/2019-04-18-component-shadowing/index.md

```diff
@@ -0,0 +1,137 @@
+---
+title: Component Shadowing
```

Comment:
I think this title could be a little more descriptive. How about `What is Component Shadowing?` 

Adding "Gatsby Themes" to it somewhere would also help SEO, but Gatsby already dominates this topic so I don't think that's important.